gradle/libs.versions.toml

```diff
@@ -20,7 +20,7 @@ event = "3.0.0"
 # Platforms
 bukkit = "1.16.5-R0.1-SNAPSHOT"
 bungeecord = "1.19-R0.1-SNAPSHOT"
-krypton = "7f7a457cdf" # Krypton uses commit-based versioning
+krypton = ""d45865ccf1" # Krypton uses commit-based versioning
```

Comment:
Error when opening and closing quotes

Reply:
That’s a merge error I will fix it when back on pc and make my next push